screen/categories/src/main/java/com/ivy/categories/CategoriesScreen.kt

```diff
@@ -556,10 +660,17 @@ private fun SelectTypeButton(
 
 @Preview
 @Composable
-private fun Preview(theme: Theme = Theme.LIGHT) {
+private fun PreviewCategoriesCompactModeEnabled(theme: Theme = Theme.LIGHT) {
```

Comment:
You need to record Paparazzi screneshots for this module. See the CI guide at the bottom of your PR description